bridge.go

```diff
@@ -353,6 +353,8 @@ func (b *bridge) SyncSourceFor(id ProjectIdentifier) error {
 	return b.sm.SyncSourceFor(id)
 }
 
+func (b *bridge) Release() { b.sm.Release() }
```

Comment:
Sorry for the delay here, I'm back from vacation :)

I can do what you ask for here, but it makes me pretty uneasy. It sounds more like either `*bridge` shouldn't try to fulfill the SourceManager interface, or Release shouldn't be in the interface. Faking the implementation by dropping in a panic here makes things pretty hard to read for an outsider. I don't really know what you mean when you say that " by definition, the bridge only exists for the duration of a solve run" or "a solve run is never responsible for closing the source manager it's using" - those are expectations of invariants that might be better checked by the compiler than through panics?

But it's your call, really! If #159 fixes this, a panic is definitely the expedient way forward.